process/templates/SIG_NAME_sandbox_stage.md

```diff
@@ -5,11 +5,23 @@
   * Link to relevant documentation.
 
 ### List of interested individuals
+The SIG have a minimum of two maintainers with different organizational affiliations.
```

Comment:
Do we really have "maintainers" for SIGs?

Reply:
+1. The SIG lifecycle doc uses the term "members", which would work fine here.